### ISSUE TITLE
Build.ps1: Use Minimal Distribution instead of Standard Distribution

### ISSUE THREAD

**Issue by merceyz** [cefsharp/cef-binary#56]
CEF issue 2169 has been resolved, this means `build.ps1` can use the `Minimal Distribution` instead of the `Standard Distribution` to speed up the download, and as a result, total build time.

https://bitbucket.org/chromiumembedded/cef/issues/2169/support-building-just-libcef_dll_wrapper

**Comment by perlun** [comment]
Good point @merceyz. Will you do it it? 😉 

I think we should also do something about the builds in general, the AppVeyor stats doesn't seem to be reporting back to this repo as it should. Would be nice to always see it red/green on a PR as usual.

**Comment by merceyz** [comment]
> I think we should also do something about the builds in general, the AppVeyor stats doesn't seem to be reporting back to this repo as it should. Would be nice to always see it red/green on a PR as usual.

Probably due to https://github.com/cefsharp/cef-binary/blob/a9fadc9c3491e923201a923093a0508ecf9c4fa5/appveyor.yml#L9

**Comment by perlun** [comment]
> Probably due to
> 
> cef-binary/appveyor.yml
> 
> Line 9 in a9fadc9
> skip_non_tags: true

Correct. I also saw it in the web UI, so I've changed it there (because it need it for another change I'm working on.)

But yeah, it should be fixed in the `.yml` as well. Feel free to fix it if you like.

**Comment by nightblade9** [comment]
Sorry to necro a seven-years-old thread, I came here to ask something similar: the binaries available are huge. If you run `dotnet publish`, the resulting folder has `interactive_ui_tests.exe` (250MB) which it looks like I can safely delete.  The next largest file is `chrome.exe` which weighs in at around 234MB.

Is it possible to switch to the minimal distribution? Soon? I'm prototyping something like Electron.NET, and I need as tiny a version of Chromium as I can get away with. (I already looked at other emeddable browsers, and didn't find anything promising.)
